### DISCUSSION TITLE
Welcome to spline-spark-agent Discussions!

### DISCUSSION THREAD

**Discussion by wajda** [AbsaOSS/spline-spark-agent#161] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alexmailajalam** [comment]
How to get end to end lineage between cross jobs?

**Reply by wajda** [reply]
What do mean by that exactly?
The first graph view on the Spline UI that you get by clicking on a execution event _is_ a cross jobs view.
Do you need something more specific?

**Reply by alexmailajalam** [reply]
job1 takes couple of inputs(a,b) and writes to c
job2 takes c and writes d

lineage shows,

1. a,b -> c
2. c ->d

I want to see end to end lineage like a,b -> c -> d

**Reply by alexmailajalam** [reply]
Never mind. Got it working. Thanks @wajda

**Reply by wajda** [reply]
it's automatic as long as `c` URIs match in both jobs. You also need to look at the later job run, because the current Spline UI version only shows the past activities, not the future, with respect to the job run you are looking at.
